parser.y

```diff
@@ -11142,6 +11151,11 @@ RoleSpecList:
 		$$ = append($1.([]*ast.UserSpec), $3.(*ast.UserSpec))
 	}
 
+BindableStmt:
+	SetOprStmt1
+|	UpdateStmt
+|	DeleteWithoutUsingStmt
```

Comment:
Just for curious. What's the impact if we ignore `DeleteWithUsingStmt`?

Reply:
We can't create SQL bind for `delete using` style statements.

Reply:
Like this SQL `DELETE FROM t1, t2 USING t1 INNER JOIN t2 INNER JOIN t3
WHERE t1.id=t2.id AND t2.id=t3.id;`?

Reply:
Yes.

Reply:
OK.